test/extended/util/prometheus/helpers.go

```diff
@@ -63,16 +64,30 @@ func waitForServiceAccountInNamespace(c clientset.Interface, ns, serviceAccountN
 	return err
 }
 
-// LocatePrometheus uses an exisitng CLI to return information used to make http requests to Prometheus.
-func LocatePrometheus(oc *exutil.CLI) (queryURL, prometheusURL, bearerToken string, ok bool) {
+// LocatePrometheus uses an existing CLI to return information used to make http requests to Prometheus.
+func LocatePrometheus(oc *exutil.CLI) (queryURL, prometheusURL, querySvcURL, prometheusSvcURL, bearerToken string, ok bool) {
```

Comment:
While I'm not aware of any consumer outside of openshift/origin that would rely on `LocatePrometheus()`, it might be more cautious to create a new function that specifically returns the "public" (e.g.  OpenShift routes) monitoring URLs.